lib/helper/AdminListHelper.php

```diff
@@ -1435,4 +1435,24 @@ public static function getUrl(array $params = array())
 	{
 		return static::getViewURL(static::getViewName(), static::$listPageUrl, $params);
 	}
+
+	/**
+	 * Кастомизация фильтра разделов
+	 * @param $filter
+	 * @return mixed
+	 */
+	protected function sectionsFilter($filter)
+	{
+		return $filter;
+	}
+
+	/**
+	 * Кастомизация фильтра элементов
+	 * @param $filter
+	 * @return mixed
+	 */
+	protected function elementsFilter($filter)
```

Comment:
Назови свое имя, демон, и скажи чего ты от меня хочешь?)
7 июня 2016 г. 7:07 PM пользователь "Nik Samokhvalov" <
notifications@github.com> написал:

> In lib/helper/AdminListHelper.php
> <https://github.com/DigitalWand/digitalwand.admin_helper/pull/27#discussion_r66101478>
> :
>
> > +	/**
> > +	 * Кастомизация фильтра разделов
> > +	 * @param $filter
> > +	 * @return mixed
> > +	 */
> > +	protected function sectionsFilter($filter)
> > +	{
> > +		return $filter;
> > +	}
> > +
> > +	/**
> > +	 * Кастомизация фильтра элементов
> > +	 * @param $filter
> > +	 * @return mixed
> > +	 */
> > +	protected function elementsFilter($filter)
>
> Раз это геттер, давай назовём getElementsFilter.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DigitalWand/digitalwand.admin_helper/pull/27/files/0ed34e387dea7d4df9b0e3ff9c90118b8350ea68#r66101478>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADKdeO_RMB4UI40eVWqbLSijkZiGxrvgks5qJZdTgaJpZM4If-lV>
> .
>